docs/variant_annotation.md

```diff
@@ -138,3 +144,34 @@ variants.
 <!-- TODO(bashir): Add some interpretation of these variants, e.g., with
 reference to sickle-cell and thalassemia. This needs to be done using other
 fields to extract phenotype information from other databases. -->
+
+### Joining with other databases
+
+You can also join with other databases that are available in BigQuery.
+For instance, there are several pathway databases that are available publicly in
+BigQuery. Here is a sample query to extract high impact variants that are in
+the double-strand break repair pathway as defined by the
+[GO (Gene Ontology) biological process (GO:0006302)](http://amigo.geneontology.org/amigo/term/GO:0006302).
+
+```
+#standardSQL
+SELECT
+  reference_name, start_position, reference_bases, ALT.alt, CSQ.*
+FROM
+  vcf_imports_external.gnomad_genomes_chr_hg19 AS T, T.alternate_bases AS ALT, ALT.CSQ AS CSQ
+WHERE
+  CSQ.SYMBOL IN (SELECT DB_Object_Symbol
```

Comment:
In general, do you know if the `SYMBOL` fields use standard/unique terms in these different datasets? IOW, are they coming from the same source?

Reply:
Hmm, okay, then please add a short comment about this in the text.

Reply:
Done.